tests/integration/helpers/iceberg_utils.py

```diff
@@ -485,7 +500,7 @@ def default_download_directory(
 
         
 def execute_spark_query_general(
-    spark, started_cluster, storage_type: str, table_name: str, query: str
+    spark, started_cluster, storage_type: str, table_name: str, query: str, additional_nodes=[]
```

Comment:
please refrain from using mutable types as default values of Python functions unless it is strictly necessary:
```python
>>> def foo(l=[]):
...     l.append(1)
...     print(l)
... 
>>> foo()
[1]
>>> foo()
[1, 1]
```

Reply:
Ouch. I did not know this. Thanks!